props/saber_fett263_buttons.h

```diff
@@ -260,7 +260,7 @@ SaberFett263Buttons() : PropBase() {}
           mss.y * mss.y + mss.z * mss.z > 70 &&
           fusor.swing_speed() < 30 &&
           fabs(fusor.gyro().x) < 10) {
-        if (millis() - push_begin_millis_ > 5) {
+        if (abs(millis() - push_begin_millis_) > 5) {
```

Comment:
no, remove all abs() and then I will review again.

Reply:
I have removed